Makefile

```diff
@@ -155,11 +147,8 @@ docs:
 	mkdocs build
 
 # NB webhook tests can only run if the controller is deployed to the cluster
-webhook-e2e-test-namespace := "webhook-e2e-test"
-
```

Comment:
The webhook tests don't actually use this namespace. Instead they create new ones from within the tests.